pkg/universe.dagger.io/examples/todoapp/plan.cue

```diff
@@ -0,0 +1,49 @@
+package todoapp
+
+import (
+	"dagger.io/dagger"
+	"universe.dagger.io/netlify"
+	"universe.dagger.io/git"
+
+	"universe.dagger.io/examples/todoapp/pkg/yarn"
+)
+
+dagger.#Plan & {
+	client: {
+		filesystem: {
+			"./_build": write: contents: actions.build.output
+			"./app/": write: contents:   actions.develop.output
+			"./app": read: {
+				contents: dagger.#FS
+				exclude: [
```

Comment:
shouldn't "./_build" be part of the `read` excludes? 

Reasoning is: 
- First time you build the app `./_build` doesn't exist so it is not sent to the build context
- Second time you build the app, `./_build` is present from your previous builds and it gets sent to the build context every time. 

I'm not sure it's something that it should be sent to the context each time since `./_build` is generally a temp dir. IMO it should be treated the same as `node_modules` as a cache withing the build.

Reply:
I don't think it's needed anymore, here is the current filesystem layout:

* `.`: the todoapp directory
* `./_build` where the build is read
* `./app` where the repository is cloned

So when reading `./app`, you don't need to exclude `_build` because it is not written to it.

Reply:
Interesting.. I'm wondering if the `yarn build` command doesn't need it as a `cache mount` to optimize something. Since I recall seeing that the output of `yarn build` usually showing a `diff` in the build output bundle changes. 

If we're not sending the `_build` folder to the context or using a cache, from the `yarn `build` perspective, build outputs will be generated entirely for every build, which IIRC it's not the same that happens when you do `yarn build` in your local machine.

Reply:
That's very possible. That said, that is already the case in the current version, and even in the current `universe.dagger.io/yarn`, so I think we should discuss that in a separate issue / PR. I don't know how to fix it TBH.

Reply:
👍 I'll open a PR for that